src/NHibernate/Multi/MultiAnyCriteriaQuery.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections.Generic;
+using NHibernate.Impl;
+using NHibernate.Loader.Criteria;
+using NHibernate.Persister.Entity;
+
+namespace NHibernate
+{
+	public partial class MultiAnyCriteriaQuery<T> : MultiAnyQueryBase<T>, IMultiAnyQuery<T>
```

Comment:
Why `IMultiAnyQuery<T>` again, since it is already brought by `MultiAnyQueryBase<T>`?